Week1/homework/app.js

```diff
@@ -2,10 +2,104 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'dont_tell_me_your_afraid', 'dont_tell_mummy', 'positive_thinking', 'time_management', 'The alchemist', 'the_power_of_now', 'ekadoli', 'Before_we_were_yours', 'hebrew_female_in_my_heart', 'amareeta'
   ];
 
-
   // Replace with your own code
-  console.log(bookTitles);
+  //  2.3
+  function makeBooksTitlesList() {
+    const ul = document.createElement('ul');
+    ul.innerHTML = 'Books Titles: ';
```

Comment:
A `ul` can only have `li`-s as children. It should not have text content. That would make for invalid HTML.